CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# Change Log
```

Comment:
is this manual at the moment ?

Reply:
Oh, good catch.. I'll add the version scripts now!